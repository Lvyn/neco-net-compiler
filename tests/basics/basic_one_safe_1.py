from snakes.nets import *

net = PetriNet('Net')
s1 = Place('s1', [4], tInteger)
s1.one_safe = True
s2 = Place('s2', [], tInteger)
s2.one_safe = True

net.add_place(s1)
net.add_place(s2)

transition = Transition('t', Expression('True'))
net.add_transition(transition)

net.add_input('s1', 't', Variable('x'))
net.add_output('s2', 't', Variable('x'))
