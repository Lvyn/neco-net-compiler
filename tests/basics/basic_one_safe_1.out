check - markings
begin marking
s1 - 
s2 - 4
end marking
begin marking
s1 - 4
s2 - 
end marking
